packages/mantine/src/components/button/Button.tsx

```diff
@@ -46,6 +46,7 @@ const _Button = forwardRef<HTMLButtonElement, ButtonProps>(
                     onClick={handleClick}
                     disabled={disabled}
                     {...others}
+                    data-loading={isLoading || loading}
```

Comment:
I am curious about the reasoning behind the order here. (why put it after others rather than before)

Reply:
No reasonning. I added it only for unit test since the change from mantine does not allow us to us the rol ```presentation``` anymore for the loading part. I don't think it is relevant to have it before or after. I will still change it to before anyway, ```{...others}``` should be last